pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +415,346 @@ def image_threshold(
+
+        The input can be real or complex data, but the output is always
+        :attr:`numpy.complex128`. The filter is fastest for images that have the power
+        of two sizes.
+
+        The filter uses a butterfly diagram for each prime factor of the
+        dimension. This makes images with prime number dimensions (i.e. 17x17)
+        much slower to compute. FFTs of multidimensional meshes (i.e volumes)
+        are decomposed so that each axis executes serially.
+
+        The frequencies of the input assume standard order: along each axis
+        first positive frequencies are assumed from 0 to the maximum, then
+        negative frequencies are listed from the largest absolute value to
+        smallest. This implies that the corners of the grid correspond to low
+        frequencies, while the center of the grid corresponds to high
+        frequencies.
+
+        Parameters
+        ----------
+        output_scalars_name : str, optional
+            The name of the output scalars. By default, this is the same as the
+            active scalars of the dataset.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            :class:`pyvista.UniformGrid` with the applied reverse FFT.
+
+        See Also
+        --------
+        fft: The direct transform.
+        low_pass: Low-pass filtering of FFT output.
+        high_pass: High-pass filtering of FFT output.
+
+        Examples
+        --------
+        Apply reverse FFT to an example image.
+
+        >>> from pyvista import examples
+        >>> image = examples.download_moonlanding_image()
+        >>> fft_image = image.fft()
+        >>> image_again = fft_image.rfft()
+        >>> image_again.point_data  # doctest:+SKIP
+        pyvista DataSetAttributes
+        Association     : POINT
+        Active Scalars  : PNGImage
+        Active Vectors  : None
+        Active Texture  : None
+        Active Normals  : None
+        Contains arrays :
+            PNGImage                complex128 (298620,)            SCALARS
+
+        See :ref:`image_fft_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageRFFT()
+        alg.SetInputDataObject(self)
+        _update_alg(alg, progress_bar, 'Performing Reverse Fast Fourier Transform.')
+        output = _get_output(alg)
+        self._change_fft_output_scalars(
+            output, self.point_data.active_scalars_name, output_scalars_name
+        )
+        return output
+
+    def low_pass(
+        self,
+        x_cutoff,
+        y_cutoff,
+        z_cutoff,
+        order=1,
+        output_scalars_name=None,
+        progress_bar=False,
+    ):
+        """Perform a Butterworth low pass filter in the frequency domain.
+
+        This filter requires that the :class:`UniformGrid` have a complex point
```

Comment:
This is another common grammatical issue we have in pyvista: "scalars" can both mean singular (as in "scalars array") and plural. I usually lean toward the plural to match the spelling of "scalars". It might be grammaticaly cleaner to use "scalars array" or something similar when using a singular.